functions/go/apply-setters/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+image: gcr.io/kpt-fn/apply-setter
+description: Apply setter values on resource fields. Setters serve as parameters for template-free setting of field values.
+categories:
+  - mutator
+imageTags:
+  - unstable
+docPath: README.md
+sourceURL: https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/apply-setter
```

Comment:
What happens to these URLs when a function is versioned?

Reply:
Re. `sourceURL`, the current plan is only show the link for each function in the catalog landing page. That means we won't show links for different versions.
Users can use this link to go to the mater and then select the desired release tag or branch.

I have shared a design doc for doc site with you offline.

I'm a little hesitant about if we should maintain `imageTags`. It can be very easily out-of-date. An alternative is to drop it in the metadata.file and gather it from git tags when building the site.